src/test/java/com/epam/validator/ValidatorTest.java

```diff
@@ -1,3 +1,65 @@
 package com.epam.validator;
 
-public class ValidatorTest { }
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+public class ValidatorTest {
+
+    @Test(expected = NullPointerException.class)
+    public void checkEmptyThrowException(){
+        String emptyString = "";
+        Validator.checkEmpty(emptyString);
```

Comment:
NullPointerException in checkMethod???????

Reply:
Done